lib/logthrdest/logthrdestdrv.c

```diff
@@ -333,6 +359,7 @@ static void
 _process_result_success(LogThreadedDestWorker *self)
 {
   _accept_batch(self);
+  exponential_backoff_reset(self->exponential_backoff);
```

Comment:
This is a double-reset here due to the function responsible for "explicit ack management".
It doesn't cause any problems, so I guess it's okay.